### DISCUSSION TITLE
Step by step configuration plex debrid 2.95

### DISCUSSION THREAD

**Discussion by djisidro2014** [itsToggle/plex_debrid#652] [General]
Good afternoon. I have followed the tutorial to the letter. I can't play any movie or TV show added to the watchlist. On the other hand, when restarting the laptop with Windows 11, the real debrid drive is unmounted. Can you help me? Thank you so much.

**Comment by demondamz** [comment]
You'll need to create a startup script that will remount the drive on reboot

**Comment by djisidro2014** [comment]
How do you create that script and what exactly do you have to put in it?
Can you give me an example? Thank you so much.

**Comment by djisidro2014** [comment]
Good afternoon. I have followed the tutorial to the letter. I can't play
any movie or TV show added to the watchlist. Can you help me? Thank you so
much.

**Comment by demondamz** [comment]
Add send me a message on discord demondamz

**Comment by djisidro2014** [comment]
I don't know how to find you. Do you have telegram? My telegram is:
https://t.me/cesareoduarte
